recipe/0103-poetry-reading-annotations/manifest1.json

```diff
@@ -0,0 +1,101 @@
+{
+  "@context":"http://iiif.io/api/presentation/3/context.json",
+  "id":"{{ id.url }}",
+  "type":"Manifest",
+  "label":{
+    "en":[
+      "Anne Sexton, Poetry Reading, 1974 -- Her Kind"
+    ]
+  },
+  "items":[
+    {
+      "id":"{{ id.path }}/canvas/1",
+      "type":"Canvas",
+      "duration":107,
+      "items":[
+        {
+          "id":"{{ id.path }}/canvas/1/paintings",
+          "type":"AnnotationPage",
+          "items":[
+            {
+              "id":"{{ id.path }}/canvas/1/painting/1",
+              "type":"Annotation",
+              "motivation":"painting",
+              "body":{
+                "id":"https://library.harvard.edu/poetry/audio/listeningbooth//PS3537E915A6x1974/Her_Kind.mp3",
+                "type":"Sound",
+                "format":"audio/mp3",
+                "duration":107
+              },
+              "target":"{{ id.path }}/canvas/1"
+            }
+          ]
+        }
+      ],
+      "annotations":[
+        {
+          "id":"{{ id.path }}/annotations.json",
+          "type":"AnnotationPage",
+          "items":[
+            {
+              "id":"{{ id.path }}/canvas/1/annotation/1",
+              "type":"Annotation",
+              "motivation":"commenting",
+              "body":{
+                "type":"TextualBody",
+                "value":"breath",
+                "format":"text/plain"
+              },
+              "target":{
+                "source":"{{ id.path }}/canvas/1",
+                "selector":{
+                  "type":"PointSelector",
+                  "t":27.660653
+                }
+              }
+            },
+            {
+              "id":"{{ id.path }}/canvas/1/annotation/2",
+              "type":"Annotation",
+              "motivation":"commenting",
```

Comment:
Per Rob's comment "Is "her kind" a transcription of the spoken content? If so then it should be motivation supplementing, not commenting. See: https://iiif.io/api/presentation/3.0/#35-values"

Listening to the recording it is a transcription so this should be:
```
"motivation":["supplementing"]
```

Reply:
We're going to change "her kind" into "repetition of 'her kind'" -- it's meant as commentary on the structure of the poem, rather than a direct transcription.  That will clarify it.

Reply:
Sorry I'm wrong here it should be a string...